libraries/radspreadprocessing/working-with-rows-and-columns/resizing.md

```diff
@@ -236,3 +237,30 @@ __Example 8__ shows how to convert and set from MS Excel to pixel column width.
 {{endregion}}
 
 The row height in MS Excel is measured in points so in order to set them you can convert this unit and set the exact number you are passing to the **SetHeight**() method for the height using the UnitHelper class.
+
+
+## LayoutHelper class
+
+The **LayoutHelper** class exposes the following methods that help to calculate the size of the page content:
+
+* **CalculateCellContentSize**: Calculates the size of the cell content.
+
+__Example 9__ shows how to get the size of the cell content.    
+
+#### __[C#] Example 9: Get the Size of the cell content__
+
+{{region cs-radspreadprocessing-working-with-rows-and-columns-resizing_9}}
+
+	Size cellContentSize = LayoutHelper.CalculateCellContentSize(worksheet, rowIndex, columnIndex);
+{{endregion}}
+
+* **CalculateCellLayoutBox**: Calculates the cell layout box.
+
+__Example 10__ shows how to get the cell layout box.    
```

Comment:
This sentence doesn't provide much more information than the caption of the snippet, so we can remove it.